packages/terra-form/src/components/MultiChoiceField.jsx

```diff
@@ -0,0 +1,96 @@
+// eslint-disable react/jsx-boolean-value
+
+import React, { PropTypes } from 'react';
+
```

Comment:
Or did you not include it because its using the Field Component? If that is the case, then you added `import 'terra-base/lib/baseStyles'` to the ChoiceFeild component